### PR TITLE
textsubs.lic: Add Ranger pace messaging.

### PR DESCRIPTION
**Pull request by Sarvatt**
It indicates if a room is wilderness, neutral, or urban for bonus purposes.

### CODE REVIEW

**Comment by Sarvatt** [comment]
Happy to bikeshed on the messaging if anyone thinks something else would be better.

File: textsubs.lic [hunk 1]
```diff
@@ -458,6 +458,13 @@ TextSubs.add('^After a moment, you see a vision of your god who calls to you by
 TextSubs.add('^After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, (\w+)\.\s+Go forth and continue your works, and you shall only attain a greater level of purity\."', 'After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, \1.  Go forth, and continue your works and you shall only attain a greater level of purity." (15/16)')
 TextSubs.add("^After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant sana'ati tree\.", "After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant Sana'ati tree. (16/16)")
 
+TextSubs.add('You glance about the area while pacing back and forth, like a large, stalking mountain cat.', 'You glance about the area while pacing back and forth, like a large, stalking mountain cat. (extreme wilderness)')
+TextSubs.add('You glide around the area like a crocodile, truly in your element.', 'You glide around the area like a crocodile, truly in your element. (slight wilderness)')
```
Comment (rpherbig) [inline textsubs.lic]: Maybe `moderate(ly)`?

File: textsubs.lic [hunk 2]
```diff
@@ -458,6 +458,13 @@ TextSubs.add('^After a moment, you see a vision of your god who calls to you by
...
+TextSubs.add('You pace back and forth, smiling slightly to yourself at the refreshing familiarity of the area.', 'You pace back and forth, smiling slightly to yourself at the refreshing familiarity of the area. (slight wilderness)')
+TextSubs.add('You pace back and forth.', 'You pace back and forth. (neutral)')
+TextSubs.add('You pace back and forth, scowling slightly at your surroundings.', 'You pace back and forth, scowling slightly at your surroundings. (slighty urban)')
```
Comment (rpherbig) [inline textsubs.lic]: Should this be `slightly urban`?

File: textsubs.lic (hunk 1 above)
Comment (Sarvatt) [reply textsubs.lic]: I do like that better. I was hung up on what the wiki called it

**Comment by Sarvatt** [comment]
changed them to extremely wilderness, moderately wilderness, neutral, moderately urban, extremely urban. Still looks a little goofy to me, but I can't think of anything better :) moderate wilderness sounds better, but moderate urban sounds worse.

File: textsubs.lic [hunk 3]
```diff
@@ -458,6 +458,13 @@ TextSubs.add('^After a moment, you see a vision of your god who calls to you by
 TextSubs.add('^After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, (\w+)\.\s+Go forth and continue your works, and you shall only attain a greater level of purity\."', 'After a moment, you see a crystal-clear vision of your god who speaks slowly and deliberately, "Your unwavering faith and devotion pleases me greatly, \1.  Go forth, and continue your works and you shall only attain a greater level of purity." (15/16)')
 TextSubs.add("^After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant sana'ati tree\.", "After a moment, you feel a clear presence like a warm blanket covering you beneath the shade of a giant Sana'ati tree. (16/16)")
 
+TextSubs.add('You glance about the area while pacing back and forth, like a large, stalking mountain cat.', 'You glance about the area while pacing back and forth, like a large, stalking mountain cat. (extremely wilderness)')
```
Comment (rpherbig) [inline textsubs.lic]: The `.`s in the "matcher" strings need to be escaped (`\.`).

**Comment by Sarvatt** [comment]
Yeah this is broken because of the duplicated 'You pace back and forth' messages, sorry about that. Testing it in a neutral room would have helped..

You pace back and forth. (neutral) smiling slightly to yourself at the refreshing familiarity of the area. (moderately wilderness)

File: textsubs.lic (hunk 3 above)
Comment (Sarvatt) [reply textsubs.lic]: escaping the .'s seems to have fixed it, sorry about that.

You pace back and forth, smiling slightly to yourself at the refreshing familiarity of the area. (moderately wilderness)

**Comment by Sarvatt** [comment]
It would be better to leave the neutral "You pace back and forth." off actually, since non rangers get that message and neutral isn't really important to know.

**Comment by Sarvatt** [comment]
Should be all fixed up now. Neutral message is removed and the .'s are escaped. Sorry for all the churn on something that should have been easy :)

**Comment by rpherbig** [comment]
Good point on the 'neutral' message. That said, I guarantee someone is going to eventually notice it's missing and try to add it back in. Could you leave a comment in the middle explaining what you just said?

And could you add a comment at the top of the block to note what these subs are for? It's something we haven't been in the habit of, but I think add some clarity to the file.

**Pull request merged**